src/components/marker/timeline-preview.tsx

```diff
@@ -160,35 +185,34 @@ export class TimelinePreview extends Component<TimelinePreviewProps> {
     };
   }
 
-  private _renderArrowButton() {
-    const isNavigationPluginOpen = this.props.isNavigationPluginOpen();
-    return (
-      <A11yWrapper onClick={this.props.onArrowClick}>
-        <button
-          aria-label={isNavigationPluginOpen ? this.props.hideNavigationTranslate : this.props.showNavigationTranslate}
-          className={[styles.markerLink, this.props.isSmallPlayer ? styles.smallPlayer : ''].join(' ')}
-          onMouseDown={this._handleMouseDown}
-          data-testid="previewArrowButton">
-          {isNavigationPluginOpen ? <Icon size={IconSize.medium} name={'arrowClose'} /> : <Icon size={IconSize.medium} name={'arrowOpen'} />}
-        </button>
-      </A11yWrapper>
-    );
-  }
-
   private _renderSmallPlayerHeader(relevantChapter: Chapter | undefined, data: any) {
     const {quizQuestions, hotspots, answerOnAir} = data;
-    const titleClassName = [styles.titleWrapper, this.props.isSmallPlayer ? styles.smallPlayer : ''].join(' ');
-    const itemsWrapperClassName = [styles.itemsWrapper, this.props.isSmallPlayer ? styles.smallPlayer : ''].join(' ');
+    const titleClassName = [styles.titleWrapper, this.props.isExtraSmallPlayer ? styles.xsPlayer : ''].join(' ');
```

Comment:
parent element already has `styles.xsPlayer` class, no need duplicate it for title or itemWrapper classnames.
```
.header {
...
  &.xs-player {
    ...
    .items-wrapper {
      ... styles for sx-size... 
      .title-wrapper {
        ... styles for sx-size... 
      }
    }
  }
  .items-wrapper {
    ... regular styles ...
  }
  .title-wrapper {
    ... regular styles ...
  }
}
```

Reply:
done